lib/common/validate.go

```diff
@@ -6,21 +6,20 @@ import (
 
 	"github.com/golang/glog"
 	"gopkg.in/go-playground/validator.v8"
+	"github.com/projectcalico/libcalico/lib/selector"
 )
 
 var validate *validator.Validate
-var nameRegex *regexp.Regexp
-var actionRegex *regexp.Regexp
-var protocolRegex *regexp.Regexp
+
+// [smc] why not define these inline to save typing on the types?
```

Comment:
Funny i thought I'd tried that but I guess not.   For some reason I didn't think it let me do that so that's why I had the init.... glad to know it works.